pathauto.module

```diff
@@ -310,7 +310,7 @@ function pathauto_entity_base_field_info_alter(&$fields, EntityTypeInterface $en
  */
 function pathauto_entity_base_field_info(EntityTypeInterface $entity_type) {
   // @todo: Make this configurable and/or remove if
-  //   https://drupal.org/node/476294 is resolved.
+  // https://drupal.org/node/476294 is resolved.
```

Comment:
this was correct actually, the comment belongs to the @todo and should be 2 spaces indented. coder is wrong there.